content/docs/components-and-props.md

```diff
@@ -78,28 +77,28 @@ ReactDOM.render(
 
 [](codepen://components-and-props/rendering-a-component)
 
-Let's recap what happens in this example:
+चलिए देकते है इस उदाहरण में क्या होता है:
 
-1. We call `ReactDOM.render()` with the `<Welcome name="Sara" />` element.
-2. React calls the `Welcome` component with `{name: 'Sara'}` as the props.
-3. Our `Welcome` component returns a `<h1>Hello, Sara</h1>` element as the result.
-4. React DOM efficiently updates the DOM to match `<h1>Hello, Sara</h1>`.
+1. हम `<Welcome name="Sara" />` के द्वारा `ReactDOM.render()` को call karte है|
+2. React `Welcome` कौम्पोनॅन्ट को `name: 'Sara'}` के साथ props के रूप मे call करती है|
+3. हमारा `Welcome` कौम्पोनॅन्ट परिणाम के तौर पर वापस करता है `<h1>नमस्ते, Sara</h1>` एलिमेंट|
+4. React DOM कुशलता पूर्वक DOM को `<h1>नमस्ते, Sara</h1>` से अपडेट करता है|
 
->**Note:** Always start component names with a capital letter.
+>**ध्यान दें:** हमेशा रेंकौम्पोनॅन्टस की नाम की शुरुवात कैपिटल लेटर से होनी चाहिए|
```

Comment:
spelling mistake रेंकौम्पोनॅन्टस

Reply:
should be reworded as component ke naam ki shuruat hamesha capital letter se honi chahiye

Reply:
3. should be हमारा `Welcome` कौम्पोनॅन्ट परिणाम के तौर पर `<h1>नमस्ते, Sara</h1>` एलिमेंट वापस lautata है